pkgs/development/python-modules/seasonal/default.nix

```
@@ -0,0 +1,51 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, setuptools
+, pytest-runner
+, numpy
+, scipy
+, pandas
+, matplotlib
+, pytestCheckHook
+, pythonOlder
+}:
+
+buildPythonPackage rec {
+  pname = "seasonal";
+  version = "0.3.1";
+  format = "setuptools";
+
+  disable = pythonOlder "3.6";
+
+  src = fetchFromGitHub {
+    owner = "welch";
+    repo = "seasonal";
+    # There are no tags or releases, but this commit corresponds to the 0.3.1 version
+    # PyPI project contains only a wheel
+    rev = "2a2396014d46283d0c7aff34cde5dafb6c462c58";
+    hash = "sha256-8YedGylH70pI0OyefiS1PG1yc+sg+tchlgcuNvxcNqE=";
+  };
+
+  nativeBuildInputs = [
+    setuptools
+    pytest-runner
+  ];
+
+  propagatedBuildInputs = [
+    numpy
+    scipy
+    pandas
+    matplotlib
+  ];
```

```suggestion
  ];
  
  passthru.optional-dependencies = {
    csv = [
      pandas
    ];
    plot = [
      matplotlib
    ];
  };
```
https://github.com/welch/seasonal/blob/2a2396014d46283d0c7aff34cde5dafb6c462c58/setup.py#L38-L44